rpcs3/Emu/Io/midi_config_types.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <string>
+
+static constexpr usz max_midi_devices = 3;
+
+enum class midi_device_type
+{
+	guitar,
+	guitar_22fret,
+	keyboard
```

Comment:
since you seem to have changed the default, you should also move the keyboard to the top